### ISSUE TITLE
phantom: process_exit is not an object on cljs 1.9.854

### ISSUE THREAD

**Issue by lsenta** [bensu/doo#142]
I get this error when doing a `lein doo phantom once test` with the last / current version of cljs.

```
Ran 2 tests containing 3 assertions.
0 failures, 0 errors.
WARNING: doo's init function was not set
#object[TypeError TypeError: undefined is not an object (evaluating 'process_exit.call')]
TypeError: undefined is not an object (evaluating 'process_exit.call')

  phantomjs://code/phantom4974545102464604389.js:81 in onError
Subprocess failed
```

**Comment by dmcgillen** [comment]
See: https://github.com/bensu/doo/pull/141

**Comment by MatthewDarling** [comment]
With #141 merged, this is probably resolved. It fixed the problem for me, anyway.

@lsenta could you give the latest `master` a try? To do that, clone the repo and run `lein install` first in the `library` directory, then again in the `plugin` directory. Then use `0.1.8-SNAPSHOT` as your version for `lein-doo`.

**Comment by miikka** [comment]
I've now released doo 0.1.8 which should fix this, so I'm closing this issue.
